src/pages/EmployeeOverview/components/EmployeeCard.jsx

```diff
@@ -0,0 +1,129 @@
+import InventoryOutlinedIcon from "@mui/icons-material/InventoryOutlined";
+import PendingActionsOutlinedIcon from "@mui/icons-material/PendingActionsOutlined";
+import {
+  Avatar,
+  Card,
+  Grid,
+  IconButton,
+  Link,
+  List,
+  ListItem,
+  Typography,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import { Box } from "@mui/system";
+import PropTypes from "prop-types";
+import LinearProgressWithLabel from "../../../components/LinearProgressWithLabel";
+
+function EmployeeCard(props) {
+  const theme = useTheme();
+  return (
+    <Card
+      sx={{
+        backgroundColor: theme.palette.primary.light,
+        padding: theme.spacing(2),
+      }}
+    >
+      <Grid container spacing={2}>
+        <Grid item xs={4}>
+          <Box>
+            <Avatar
+              variant="square"
+              sx={{ width: "100%", height: "auto", maxWidth: "125px" }}
+              src={props.employee.profileImageUrl}
+            />
+            <Box
+              sx={{
+                display: "flex",
+              }}
+            >
+              <IconButton>
+                <InventoryOutlinedIcon />
+                <Typography variant="body2">
+                  {props.employee.finishedCourses || 0}
+                </Typography>
+              </IconButton>
+              <IconButton>
+                <PendingActionsOutlinedIcon />
+                <Typography variant="body2">
+                  {props.employee.ongoingCourses || 0}
+                </Typography>
+              </IconButton>
+            </Box>
+          </Box>
+        </Grid>
+        <Grid item xs={7}>
+          <List disablePadding>
+            <ListItem disablePadding alignItems="flex-start">
+              <Typography variant="h6" noWrap>
+                {props.employee.name || "No Name"}
+              </Typography>
+            </ListItem>
+            <ListItem disablePadding>
+              <Typography variant="body2" noWrap>
+                {props.employee.jobTitle || "No Job Title"}
+              </Typography>
+            </ListItem>
+            <ListItem disablePadding>
+              <Typography variant="body2" noWrap>
+                {props.employee.email || "No Email"}
+              </Typography>
+            </ListItem>
+            {props.employee?.courses?.[0] ? (
+              <ListItem disablePadding>
+                <Typography variant="body1" marginTop={2}>
+                  Courses
+                </Typography>
+              </ListItem>
+            ) : (
+              <Typography variant="body1" marginTop={2}>
+                No Courses Assigned
+              </Typography>
+            )}
+            {props.employee?.courses?.[0] ? (
+              <ListItem disablePadding>
+                <LinearProgressWithLabel
+                  title={props.employee.courses[0].name}
+                  value={props.employee.courses[0].progress}
```

Comment:
No default values

Reply:
Not needed, because the LinearProgress component has default values by its own. You don't have to pass them